### PR TITLE
add neon instruction abs for floating-point

### PR DESCRIPTION
**Pull request by surechen**
add neon instruction vabs_f32 vabsq_f32  vabs_f64 vabsq_f64

### CODE REVIEW

**Comment by Amanieu** [comment]
Have you looked at adding this using `stdarch-gen` instead? It auto-generates code from `crates/stdarch-gen/neon.spec`.

**Comment by surechen** [comment]
> Have you looked at adding this using `stdarch-gen` instead? It auto-generates code from `crates/stdarch-gen/neon.spec`.

Hi，Thanks for reviewing this. I will read and edit this pr.

**Comment by surechen** [comment]
> Have you looked at adding this using `stdarch-gen` instead? It auto-generates code from `crates/stdarch-gen/neon.spec`.

Done. Thanks.

File: crates/stdarch-gen/neon.spec [hunk 1]
```diff
@@ -166,6 +166,29 @@ validate TRUE, FALSE, FALSE, FALSE
...
+
+/// Floating-point absolute value
+name = vabs
+fn = simd_fabs
+a =  -0.1, -2.2, -3.3, -6.6
+validate 0.1, 2.2, 3.3, 6.6
+
+arm = vabs
+aarch64 = fabs
+generate float32x2_t:float32x2_t, float32x4_t:float32x4_t
```
[the remark this answers is not in the document]
Comment (surechen) [reply crates/stdarch-gen/neon.spec:182]: Done. Thanks.

**Pull request merged**